approximations.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from scipy.special import jn  # Importing the j0 function
+
+# Define constants and data arrays
+R = 4 #number of resources
+out = 10 #length of output samples
+phase_shift = np.array([0.01, 0.1, 0.3, 0.5, 0.7, 0.9])
+P_1 = np.array([0.19672402619617100000, 0.17944352000247800000, 0.16146550626352600000,
+                0.11949720031367500000, 0.06226986498344590000, 0.02995015788875120000]) #enter P_1 values
+
+
+def sinc(x):
+    if x == 0:
+        return 1.0
+    else:
+        return np.sin(x) / x
```

Comment:
Is it possible to just use [this](https://numpy.org/doc/stable/reference/generated/numpy.sinc.html) directly?

Reply:
The values produced by sinc(x) were different because this function computes sin(pix)/(pix), as opposed to the mathematical expression sin(x)/x.